tests/Unit/Common/Internal/ValidateTest.php

```diff
@@ -100,14 +100,22 @@ public function testIsTrueWithFalse()
         Validate::isTrue(false, Resources::EMPTY_STRING);
     }
 
-    public function testIsDateWithDate()
+    public function testIsDateWithDateTime()
     {
         $date = Utilities::rfc1123ToDateTime('Fri, 09 Oct 2009 21:04:30 GMT');
         Validate::isDate($date);
 
         $this->assertTrue(true);
     }
 
+    public function testIsDateWithDateTimeImmutable()
+    {
+        $date = new DateTimeImmutable();
```

Comment:
Please change to `\DateTimeImmutable` instead of `DateTimeImmutable`.

Reply:
Nice catch, thanks! Changed.